README.md

```diff
@@ -11,3 +12,16 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
+
+$O(\log_{2} n) = \exists c, n_0: T(n) \le c\log_{2}(n) \forall n \ge n_0$   {big-oh where $f(n) = \log_{2}(n)$}
```

Comment:
This isn't a notation we've used -- please use the definitions we used in class.

Reply:
My apologies, I fixed the notational issues with my proof. If there is something still wrong with the definition of big-oh where f(n) = log_{2}(n) or f(n) = log_{5}(n), please let me know and I'll see what I can do about correcting it.